### DISCUSSION TITLE
CSS doesn't compile in subfolders of theme

### DISCUSSION THREAD

**Discussion by MCoreServices** [tailpress/tailpress#198] [Q&A]
Hi there,

This is my first time developing with Tailpress and I'm loving it so far. I'm currently building a WooCommerce-project and I noticed that CSS doesn't compile when adding classes to the Woocommerce templates. As you can see in [this screenshot](https://capture.dropbox.com/Og6ruW9jESzQQOeT), I'm trying to display a grid with 4 columns, but to no avail.

- I tried deleting node_modules and reinstalling
- I tried rebuilding Lando (I use it as a local environment)
- I tried running dev/watch-poll

My folder structure: https://capture.dropbox.com/AouZkiGn0NpYEFSa

Snippet from tailwind.config.js (I added the WooCommerce-folder as a last try):
```
module.exports = {
    content: [
        './*.php',
        './**/*.php',
        './woocommerce/**/*.php',
        './woocommerce/*.php',
        './resources/css/*.css',
        './resources/js/*.js',
        './safelist.txt'
    ],
    etc. ...
```

Nothing seems to work to make it compile properly. Do any of you have some input for me please?

**Comment by jeffreyvr** [comment]
Your config looks good. Maybe this is caching related?

If you haven't already, you might try and set your environment type to development in `wp-config.php`, your stylesheet will be loaded without being browser-cached.

```php
define('WP_ENVIRONMENT_TYPE', 'development');
```
